utils/exporters/blender/addons/io_three/constants.py

```diff
@@ -329,3 +329,4 @@
 
 DOUBLE_SIDED = 'doubleSided'
 
+EXPORT_SETTINGS_KEY = 'three_export_settings'
```

Comment:
Normally I lean towards snake_case as much as possible too. Since many of those constants are passed through to `three` in the JSON file it felt like a necessary evil to match case convention of `three`. Thanks for conforming.